src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -70,6 +70,14 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
                     response = new HttpResponseMessage(statusCode);
                     response.Content = new StringContent(body);
 
+                    // we default the Content-Type here, but we override below with any
+                    // Content-Type header the user might have set themselves
+                    if (Utility.IsJson(body))
```

Comment:
Sounds good. I just wanted to bring this up as I worry about forcing the content type to be application/json without considering what the client specified in the request. This would make it impossible for clients that wish to have the content type set to something else (regardless of the payload format) to do so. I guess it's no worse than the previous behavior since it was probably always using text/plain.